region/tourian/main.json

```diff
@@ -3876,13 +3879,43 @@
                   "notable": false,
                   "requires": [
                     {"or": [
-                      "SpaceJump",
-                      "canConsecutiveWalljump"
+                      "canCarefulJump",
+                      "ScrewAttack",
+                      "Morph",
+                      {"enemyDamage": {
+                        "enemy": "Tourian Space Pirate (all)",
+                        "type": "contact",
+                        "hits": 1
+                      }}
+                    ]},
+                    {"or": [
+                      "canCarefulJump",
+                      "ScrewAttack",
+                      "HiJump"
+                    ]},
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      {"and": [
+                        "SpaceJump",
+                        {"or": [
+                          "HiJump",
+                          {"and": [
+                            "canCarefulJump",
```

Comment:
is this doable without acid frames with space jump and no wall jumps?

Reply:
With min maxing you could.  It's probably better to have the acid frames here.  Until there is a `canOptimizeMovement` tech.